src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/entity/DynamicFrameworkContract.kt

```diff
@@ -31,4 +31,6 @@ data class DynamicFrameworkContract(
   val maximumAge: Int?,
   @NotNull val allowsFemale: Boolean,
   @NotNull val allowsMale: Boolean,
+
+  var contractReference: String,
```

Comment:
Could we add this to the CRS report too, please?

https://github.com/ministryofjustice/hmpps-interventions-service/blob/main/helm_deploy/hmpps-interventions-service/reports/crs_performance_report.sql#L5

```diff
-    'coming-later'          AS crs_contract_reference,
+    r.contract_reference    AS crs_contract_reference,
```